src/customHooks/useTodoChange.tsx

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable max-len */
+import { useState } from 'react';
+
+export const useTodoChange = (): [number | number[], boolean, ((changingTodosId: number | number[], state: boolean) => void)] => {
```

Comment:
custom hook - great 👏🏼 
just multi-line types string for readability
```suggestion
export const useTodoChange = (): [
	number | number[],
	boolean,
	((changingTodosId: number | number[], state: boolean) => void)
] => {
```